src/style.css

```diff
@@ -0,0 +1,141 @@
+* {
+  border: none;
+  outline: none;
+}
+
+@font-face {
+  font-family: "Awenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Awenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 800;
+}
+
+/* Both search forms styles */
+
+body {
+  box-sizing: border-box;
+  border: none;
+  outline: none;
+}
+
+.container {
+  margin-top: 20px;
+}
+
+.button {
+  cursor: pointer;
+  position: absolute;
+  background-color: transparent;
+}
+
+.input {
+  font-family: "Awenir", sans-serif;
+  color: #3d4e61;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+}
+
+/* Smooth fade out placeholder */
+
+/* .input:focus::placeholder {
+  opacity: 0;
+  transition: opacity 0.4s ease;
+} */
+
+/* Large search form styles  */
+
+.large__container {
+  height: 70px;
+}
+
+.large__button {
+  width: 62px;
+  height: 70px;
+}
+
+.large__image {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.large__input {
+  padding-left: 62px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 800;
+}
+
+.large__input:focus {
+  font-weight: 800;
+}
+
+.large__input:focus::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+/* Small search form styles  */
+
+.small__container {
+  height: 42px;
+}
+
+.small__button {
+  padding: 0;
+  width: 33px;
+  height: 42px;
+}
+
+.small__image {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.small__input {
+  padding-left: 33px;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  font-weight: 900;
+}
+
+.small__input:focus {
+  font-weight: 900;
+}
+
+.small__input::placeholder {
+  font-weight: 400;
+}
```

Comment:
Shouldn't it be same with large? Wouldn't it be more logical to set it for `input` instead?